cmd/adsysd/integration_tests/systemdaemons/vfs_mocks.py

```diff
@@ -85,16 +85,32 @@ def Mounted(self, bus_name: str, obj_path: dbus.ObjectPath, display_name: str, s
     def get_spec_values(self, protocol: str, mount_spec: tuple[list[bytes], dict[str, list[bytes]]]) -> tuple[str, str]:
         server, share = '', ''
 
+        # Mount spec for smb-share when mounting smb://example.com/mount/path
+        # mount_spec [
+        #  "/path" + \0,
+        #  dict {
+        #    "server": "example.com" + \0,
+        #    "share":  "mount" + \0,
+        #     "type":  "smb-share" + \0,
+        #  }
+        # ]
         if protocol == 'smb-share':
             server = ''.join([dbus.String(x) for x in mount_spec[1].get("server")])
             share = ''.join([dbus.String(x) for x in mount_spec[1].get("share")])
 
+        # Mount spec for nfs when mounting nfs://example.com/mount/path
+        # mount_spec [
+        #  "/mount/path" + \0,
+        #  dict {
+        #    "host": "example.com" + \0,
+        #    "type": "nfs" + \0,
+        #  }
         elif protocol == 'nfs':
             server = ''.join([dbus.String(x) for x in mount_spec[1].get("host")])
             share = ''.join([dbus.String(x) for x in mount_spec[0]])
 
-        # Removes the null characters from the string
-        share = share.replace('\0', '')
-        server = server.replace('\0', '')
+        # Removes the trailing null characters from the strings
+        share = share.strip('\0')
```

Comment:
No need for argument I think

Reply:
Yes, we do.The python documentation states "The chars argument is a string specifying the set of characters to be removed. If omitted or None, the chars argument defaults to removing whitespace."